runtime/rasdump/dmpagent.c

```diff
@@ -299,7 +299,7 @@ static const J9RASdumpSpec rasDumpSpecs[] =
 		  NULL,
 #endif /* defined(WIN32) */
 		  NULL,
-		  0,
+		  1,
```

Comment:
Why does the priority need to change here?

Reply:
see https://github.com/eclipse/openj9/pull/7406#issuecomment-539719842